client/src/store/index.js

```diff
@@ -0,0 +1,14 @@
+import { configureStore } from '@reduxjs/toolkit';
+import rootReducer from './reducers/index';
+import { PERSIST, PURGE } from 'redux-persist';
+export const store = configureStore({
+  reducer: rootReducer,
+
+  // PlainObject 이슈제거
+  middleware: (getDefaultMiddleware) =>
+    getDefaultMiddleware({
+      serializableCheck: {
+        ignoreActions: [PERSIST, PURGE],
+      },
```

Comment:
```suggestion
      serializableCheck: {
        ignoreActions: [PERSIST, PURGE],
      },
```

Persist 시 non-serializable object injection 발생하여 추가.